lib/unit-ruby/deposit_account.rb

```diff
@@ -24,5 +24,51 @@ class DepositAccount < APIResource
     include ResourceOperations::Create
     include ResourceOperations::Save
     include ResourceOperations::Find
+
+    def resource_path
+      self.class.resource_path(id)
+    end
+
+    def close (reason)
```

Comment:
Nit: usually there's no space between the method name and arguments when using parenthesis. Ruby also allows `def close reason` but that's not generally the style we follow.

```suggestion
    def close(reason)
```